pc2ccs/pc2v9
# I784 - CLICS compliant contest.yaml properties added, fixes #784

### Description of what the PR does
Allows PC2 to read and accept CLICS compliant contest.yaml files.
Adds CLICS 2022-07 (and draft/2023-06) contest.yaml properties and code to look for CLICS contest.yaml properties first.  If not found, falls back to the older undocumented yaml properties.
This PR allows the use of WF style contest.yaml files without manual modification and maintaining separate copies.

### Issue which the PR addresses
Fixes #784 

### Environment in which the PR was developed (OS,IDE, Java version, etc.)
Windows 11
Eclipse Version: 2021-12 (4.22.0)
Build id: 20211202-1639
Java JDK 1.8.0_321

### Precise steps for _testing_ the PR
1) Start with a clean setup (no existing profiles)
2) Start a PC2 server --load clics_sumithello
3) Note that the contest correctly loads by visiting the Settings tab of the administrator and checking the contest name. Also visit the times tab to confirm contest times are correct.

To confirm that old contest.yaml files load correctly, simply load the sumithello sample contest from scratch.  Note the settings loaded correctly.
